youths/schema.py

```diff
@@ -122,6 +126,41 @@ class CreateMyYouthProfileInput(YouthProfileFields):
     )
 
 
+class CreateYouthProfileMutation(relay.ClientIDMutation):
+    class Input:
+        service_type = graphene.Argument(AllowedServiceType, required=True)
+        profile_id = graphene.Argument(graphene.ID, required=True)
+        youth_profile = CreateMyYouthProfileInput(required=True)
+
+    youth_profile = graphene.Field(YouthProfileType)
+
+    @classmethod
+    @staff_required(required_permission="manage")
+    @transaction.atomic
+    def mutate_and_get_payload(cls, root, info, **input):
+        input_data = input.get("youth_profile")
+        if input.get("service_type") != ServiceType.YOUTH_MEMBERSHIP.value:
```

Comment:
hm, wouldn't `input.get("service_type")` be `YOUTH_MEMBERSHIP` whereas `ServiceType.YOUTH_MEMBERSHIP.value` is `youth_membership`?

Reply:
You could think that, but actually `type(input.get("service_type")) == <class 'str'>` and in this example `input.get("service_type") == 'youth_membership'` 👀

Reply:
no, yeah, I expected it to be a string, but I thought it would be all uppercase 🤔 hm, odd. but okay 😅

Reply:
```
class ServiceType(Enum):
    HKI_MY_DATA = "hki_my_data"
    BERTH = "berth"
    YOUTH_MEMBERSHIP = "youth_membership"
    GODCHILDREN_OF_CULTURE = "godchildren_of_culture"

    class Labels:
        HKI_MY_DATA = _("Helsinki My Data")
        BERTH = _("Berth")
        YOUTH_MEMBERSHIP = _("Youth Membership")
        GODCHILDREN_OF_CULTURE = _("Godchildren of Culture")
```

Reply:
yeah, I know how the enum is declared. but it's weird that directly getting `input.get("service_type")` does not give you the uppercase string, as graphene's enums are represented. nevermind 🙂 if this works, then this is ok.